### DISCUSSION TITLE
Looking for an ETA

### DISCUSSION THREAD

**Discussion by mikelaff72** [argoproj-labs/argocd-operator#852] [Q&A]
Wondering when this version (2.6.1) will be pushed out and available?

**Comment by mikelaff72** [comment]
following up...

**Comment by joyartoun** [comment]
Hi, I am also very curious about this. I see that the latest release of argocd is version 2.6.4. when will we see a release of argocd operator that will include that version? Been a long time since it was officially released (Dec 2022). I don't see anything in the roadmap either.

@iam-veeramalla @reginapizza if you have any info, please share.

**Reply by iam-veeramalla** [reply]
> Hi, I am also very curious about this. I see that the latest release of argocd is version 2.6.4. when will we see a release of argocd operator that will include that version? Been a long time since it was officially released (Dec 2022). I don't see anything in the roadmap either.
> 
> @iam-veeramalla @reginapizza if you have any info, please share.

Hi, we will have a release this week, which will include Argo CD version 2.6.3.

**Reply by joyartoun** [reply]
Thanks @iam-veeramalla , will the updated version come out to OLM as soon as it's released as well? Don't know how that works

**Reply by iam-veeramalla** [reply]
> Thanks @iam-veeramalla , will the updated version come out to OLM as soon as it's released as well? Don't know how that works

Yes 👍

**Reply by joyartoun** [reply]
@iam-veeramalla still no release?

**Reply by reginaelyse** [reply]
@joyartoun I'm starting to work on it in our new sprint starting today so it should be out sometime next week :)
